src/parser/core/modules/Procs.tsx

```diff
@@ -20,31 +20,26 @@ const ICON_MUSCLE_MEMORY = 1994
 
 export interface ProcBuffWindow {
 	start: number,
-	stop?: number
+	stop: number, // All windows will eventually be closed with stopAndSave, either due to a remove event or EOF.  Force not-null to make reliable for consumers and initialize with stop = start
```

Comment:
`stop = start` is technically invalid (a consumer needs to know that the `stop` value could be incorrect, but unlike nullable, does not force them to check it).

in saying this; there already seems to be a separation between current and historical - possibly worth defining

```ts
interface CurrentWindow {
	start: number,
	// ...
}

interface HistoricalWindow extends CurrentWindow {
	end: number
}
```

this would allow the current- logic to operate without encoding an `end` time, but still ensure that code working with historical data has a guaranteed `end`.